CheckerUI_xforms/CheckerUI/CheckerUI/CheckerUI/Helpers/Order/OrderItemView.cs

```diff
@@ -26,15 +33,12 @@ public class OrderItemView : BaseViewModel
         public Command DoneCommand { get; }
         public Command ReadyCommand { get; }
 
-        private readonly OrderItemModel m_orderItem;
-        private string m_OrderStatusText;
-
-        public OrderItemView(string i_Name, OrderButtonModel i_Button, BaseLineViewModel.OrderIDNotifier idNotifier)
+        public OrderItemView(string i_Name, OrderButtonModel i_OrderButton, OrderIDNotifier idNotifier)
         {
-            m_Button = new OrderButtonModel();
-            m_Button = i_Button;
+            m_OrderButton = new OrderButtonModel();
+            m_OrderButton = i_OrderButton;
```

Comment:
if you're passing along a ref for a button either way, line 38 is redundant, no?